<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BalanceWindow</class>
 <widget class="QWidget" name="BalanceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>441</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>878</width>
    <height>441</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>878</width>
    <height>441</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Stan konta</string>
  </property>
  <widget class="QLabel" name="AccountNameLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>691</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Numer Konta:</string>
   </property>
  </widget>
  <widget class="QLabel" name="BalanceLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>691</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stan konta:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wróć</string>
   </property>
  </widget>
  <widget class="QLabel" name="MainLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>5</y>
     <width>861</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stan konta</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
